site-src/contributing/enhancement-requests.md

```diff
@@ -29,28 +29,21 @@ It is unlikely to require an enhancement if it:
 If you're unsure the proposed work requires an enhancement, file an issue
 and ask.
 
-## When to Create a New Enhancement
+## New Enhancement process
```

Comment:
```suggestion
## New Enhancement Process
```

Reply:
@youngnick I think you may have missed pushing up the latest set of updates here.

Reply:
So I did. Fixed now.